docs/breaking-app-change.md

```diff
@@ -0,0 +1,50 @@
+# Getting a breaking app change through Jenkins
+
+This guide describes the scenario where you are changing the behaviour of an
+application, and the E2E tests require changes to get the tests passing 
+against your change.
+
+1. Create a PR on the publishing-e2e-tests which makes the E2E tests pass when 
+   run against the updated version of your application(s). The
+   [how to run the tests guide][how-to-run-tests] explains how to perform this
+   on your development machine.
+
+2. Once your test works locally you’ll want to get both your application and E2E
+   tests repo changes passing the GitHub CI checks.
+
+   1. Go to the [publishing-e2e-tests project on Jenkins][jenkins-job] and click
+      through into your branch with the updated tests.
+
+   2. Using the “Build with Parameters” option on the left hand bar replace the
+      following parameters and hit the Build button.
+      
+      `${YOUR_APP_NAME}_COMMITTISH` with the commit SHA of your change
+       
+      `ORIGIN_REPO` with your application repo name, e.g. government-frontend
+      
+      `ORIGIN_COMMIT` with the same commit SHA as above
+
+   3. Publishing E2E tests uses the `ORIGIN_REPO` + `ORIGIN_COMMIT` when pushing
+      the GitHub status, and the `_COMMITTISH` to know which version of each app
+      to test against.
+
+3. Merge your application changes as you would normally.  Unless your change to
```

Comment:
Seems like this would be a step but is just background info, wondering if we could add this to the above